### DISCUSSION TITLE
Discussion: Colors in EZA

### DISCUSSION THREAD

**Discussion by cafkafk** [eza-community/eza#359] [Development]
# Discussion Thread: Colors in EZA 

As we continue to refine and enhance our beloved EZA, one of the most
intriguing aspects that **never** gets overlooked is the color scheme.

Looking forward to seeing your colorful insights and suggestions for this
vibrant topic!

*Note: This thread is dedicated to discussing the colors of EZA. For other
topics or technical issues, please refer to the relevant threads or create a
new one.*

**Comment by Solarunit** [comment]
By default owner's permissions, size, user/group is in bold text. Not sure why it should be that way by default.
I have to set EXA_COLORS to disable it.

I think that date/time color should be a bit different, because it looks very similar to folders color.

here is my EXA_COLORS I am currently using (xx=37 is added recently for dashes as workaround)

`EXA_COLORS="uu=33:gu=33:ur=33:uw=31:ux=32:ue=32:sn=32:da=36:xx=37"`

Colors are also depend on the terminal theme you're using. I found that default eza/exa colors look best when using Gotham theme in Alacritty.

![eza](https://github.com/eza-community/eza/assets/20891064/a8fd19db-d8c0-4a83-a9b4-a359317530ed)

you can see eza colors look better on the left because they a bit "dimmed" (yellow is too bright on the right).

**Comment by roadkell** [comment]
Color scales for both sizes and last modified dates, please.
Also, color of `x` in permissions should match the color of executable files, imho.
